src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+interface Props {
+  todo: Todo;
+  unselect: () => void;
+}
+
+export const TodoModal: React.FC<Props> = ({ todo, unselect }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    getUser(todo.userId)
+      .then(apiUser => {
+        setUser(apiUser);
+        setIsLoading(false);
+      });
+  }, []);
+
```

Comment:
I'm not sure, but maybe it's better to use async - await here

Reply:
async/await - just different way to work with promises, so I think it personal preferences